fatiando/gravmag/transform.py

```diff
@@ -262,6 +263,36 @@ def tga(x, y, data, shape, method='fd'):
     return res
 
 
+def tilt(x, y, data, shape):
+    """
+    Calculates the magnetic tilt, as defined by Miller and Singh (1994):
```

Comment:
Maybe not "magnetic tilt", as this can be used for other things as well? "potential field tilt angle"?